test/extended/deployments/deployments.go

```diff
@@ -1554,4 +1554,42 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
 			o.Expect(strings.TrimSpace(rcs.Items[0].Spec.Template.Spec.Containers[0].Image)).NotTo(o.BeEmpty())
 		})
 	})
+
+	g.Describe("when no triggers are defined [Conformance]", func() {
+		dcName := "paused"
+		g.AfterEach(func() {
+			failureTrap(oc, dcName, g.CurrentGinkgoTestDescription().Failed)
+		})
+
+		g.It("should manually make deployment", func() {
+			dc, err := createDeploymentConfig(oc, pausedDeploymentFixture)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			o.Expect(dc.Name).To(o.Equal(dcName))
+
+			g.By("remove all the triggers")
+			_, err = oc.Run("set").Args("triggers", "dc/"+dcName, "--remove-all").Output()
```

Comment:
you can just create a fixture with `triggers: []`

Reply:
prefer using api call unless you'd be explicitly testing cli

Reply:
Thanks , updated.